src/components/molecules/ReactionsBar/ReactionsBar.scss

```diff
@@ -0,0 +1,37 @@
+@import "scss/constants";
+
+.ReactionsBar {
+  display: flex;
+  flex: 1;
+  justify-content: space-around;
+  margin: 5px 0;
+  width: 100%;
+
+  &--mute-button {
+    display: flex;
+    align-items: center;
+    font-size: 20px;
+    cursor: pointer;
+  }
+
+  &--leave-seat-button {
+    display: inline-block;
+    font-weight: 700;
+    text-decoration: none;
+    border-radius: 8px;
+    text-align: center;
+    font-size: 0.7em;
+    transition: all 400ms $transition-function;
+    line-height: 1.5;
+    border: 1px solid transparent;
+    transform: translateY(0);
+    cursor: pointer;
+    padding: 4px 10px;
+    background: rgb(0, 0, 0);
+    color: $white;
+
+    &:hover {
+      background-color: rgba(255, 255, 255, 0.5);
```

Comment:
`constants.scss`

Reply:
fixed in 9cbbfe8f7c30d7ffb04d2dd93820d4d5add71ec4